src/webcomponents/helperfunctions/jsonToCol.ts

```diff
@@ -0,0 +1,51 @@
+import { JsonToCol, LanguagesInput } from "../../@types/interfaces";
+
+const mapLangToSheetColumns = (languages: LanguagesInput) => {
+  let columnTitle: Array<object> = [];
+  Object.keys(languages).map(
+    (title: string) =>
+      (columnTitle = [
+        ...columnTitle,
+        { type: "text", title: `${title}`, width: 150 },
+      ])
+  );
+  return columnTitle;
+};
+
+const setKeys = (languages: LanguagesInput): Array<string> => {
+  let languagesKeys: Array<string> = [];
+  Object.keys(languages).map(
+    (titles) => (languagesKeys = [...languagesKeys, titles])
+  );
+  const keyValue = Object.keys(languages[languagesKeys[0]]);
+  return keyValue;
+};
+
+const mapLangToSheetData = (languages: LanguagesInput) => {
+  let rowsData: Array<object> = [];
+  let keys: Array<string> = setKeys(languages);
+  let tempRows: any = [];
+  let tempCol: Array<object> = [];
+
+  Object.keys(languages).map(
```

Comment:
same mistake related to the syntactic sugar (check comment in `mapLangToSheetColumns`)